test/testODE.jl

```diff
@@ -23,5 +20,5 @@ for i in 1:8
 end
 params_ode = ODEoptimizer(low, upp, p, 5, g1, g2, g1_0, g2_0)
 @profile ODEoptimizer(low, upp, p, 5, g1, g2, g1_0, g2_0)
-@profile ode_plotIt(params_ode, g1, g2, g1_0, g2_0, pop, 8, "")
+@profile ode_plotIt(params_ode, g1, g2, g1_0, g2_0, pop, 8, "", false)
```

Comment:
`:false` works I guess, may I change it?

Reply:
Yes, thanks!

Reply:
Sorry, that wasn't a problem. I had merged a revert commit from my last night PR, it was failing so I reverted it this morning. I pulled master into your branch, hopefully it will pass now.